### DISCUSSION TITLE
Game freezes when generating the world.

### DISCUSSION THREAD

**Discussion by NorkovayaShubidze** [Kron4ek/minecraft-vortex-launcher#51] [Q&A]
Very much like your launcher, handy and simple, and it is possible to play on the network and add mods. However, there is one mod for Fabric (called Fabric API) so when I put the newest version of this mod, then after starting the game with your launcher during the generation of the world game just hangs because of this mod. The situation is saved by an old version of the mod Fabric API, which fits the launcher, but not all of the new mods are compatible with the old version of Fabric API. Just on other luncher newest version of this mod works without problems, but with yours, unfortunately, there are such problems (((() I want to add, when I allocate RAM game through your luncher, namely 6144 mb, then when viewing the amount of RAM taken in the game itself, it shows a smaller value than I pointed in your luncher.

**Comment by Kron4ek** [comment]
Helllo. I can't reproduce this issue, tried 1.18.2 and 1.19 with the latest Fabric API at the moment (0.55.1 and 0.55.2). What game version are you using? Also, are you on Linux or on Windows?

**Reply by NorkovayaShubidze** [reply]
I play on Windows 11 Home, I have the latest version of the JDK installed, But all versions of the Fabric API that are higher than version 0.53.3, I just have world loading crash, the game completely hangs on the world loading screen when it gets to 100%. By your nickname, I think you can speak Russian. We can certainly continue in English, but I know both languages.

**Reply by Kron4ek** [reply]
Хм, я пробовал на Linux, но может эта проблема только на Windows проявляется. Я попробую на Windows ее воспроизвести.

**Reply by Kron4ek** [reply]
Я попробовал на Windows 10 LTSC 2019 (другой версии Windows под рукой нет), и все равно эта проблема у меня не возникает. У тебя последняя версия лаунчера - 1.1.18? Попробуй скачать клиент заново через сам лаунчер и снова установить fabric-loader и Fabric API.
